### PR TITLE
first pass at renaming repo

### PR DESCRIPTION
**Pull request by kevin-ortega**
## Summary

Renaming open-liberty repo to liberty repo to more accurately reflect what the buildpack contributes. 

## Use Cases

The buildpack can provide open liberty and WebSphere liberty UBI images and we will eventually add WebSphere support to the ubuntu stack.  

## Checklist

* [x] I have viewed, signed, and submitted the Contributor License Agreement.
* [ ] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [ ] I have added an integration test, if necessary.
* [x] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [x] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# `gcr.io/paketo-buildpacks/open-liberty`
+# `gcr.io/paketo-buildpacks/liberty`
 
-The Paketo Open Liberty Buildpack is a Cloud Native Buildpack that contributes Open Liberty for Java EE support.
+The Paketo Open Buildpack is a Cloud Native Buildpack that contributes Open Liberty or WebSphere Liberty for Java EE support.
```
Comment (dmikusa-pivotal) [inline README.md:3]:
```suggestion
The Paketo Liberty Buildpack is a Cloud-Native Buildpack that contributes Open Liberty or WebSphere Liberty for Java EE support.
```

File: cmd/buildpack/main.go [hunk 2]
```diff
@@ -21,7 +21,7 @@ import (
 
 	"github.com/paketo-buildpacks/libpak"
 	"github.com/paketo-buildpacks/libpak/bard"
-	"github.com/paketo-buildpacks/open-liberty/openliberty"
+	"github.com/paketo-buildpacks/liberty/openliberty"
```
Comment (dmikusa-pivotal) [inline cmd/buildpack/main.go:24]:
```suggestion
	"github.com/paketo-buildpacks/liberty/liberty"
```

File: helper/linker.go [hunk 3]
```diff
@@ -19,9 +19,9 @@ package helper
 import (
 	"encoding/xml"
 	"fmt"
-	"github.com/paketo-buildpacks/open-liberty/internal/server"
-	"github.com/paketo-buildpacks/open-liberty/internal/util"
-	"github.com/paketo-buildpacks/open-liberty/openliberty"
+	"github.com/paketo-buildpacks/liberty/internal/server"
+	"github.com/paketo-buildpacks/liberty/internal/util"
+	"github.com/paketo-buildpacks/liberty/openliberty"
```
Comment (dmikusa-pivotal) [inline helper/linker.go:24]:
```suggestion
	"github.com/paketo-buildpacks/liberty/liberty"
```

File: helper/linker.go [hunk 4]
```diff
@@ -64,7 +64,7 @@ func (f FileLinker) Execute() (map[string]string, error) {
 
 	layerDir, ok := os.LookupEnv("BPI_OL_RUNTIME_ROOT")
 	if !ok {
-		layerDir = "/layers/paketo-buildpacks_open-liberty/open-liberty-runtime"
+		layerDir = "/layers/paketo-buildpacks_liberty/open-liberty-runtime"
```
Comment (dmikusa-pivotal) [inline helper/linker.go:67]: Not sure about this one? I don't like the fact that we have this line hard coded here. I would honestly prefer if we removed this line & just fail if `BPI_OL_RUNTIME_ROOT` is not set. That should always be set by the buildpack. I think it's OK to fail if it's not set.

File: helper/linker.go [hunk 5]
```diff
@@ -115,7 +115,7 @@ func (f FileLinker) Configure(layerDir, appDir string) error {
 
 	f.BaseLayerPath = os.Getenv("BPI_OL_BASE_ROOT")
 	if f.BaseLayerPath == "" {
-		f.BaseLayerPath = "/layers/paketo-buildpacks_open-liberty/base"
+		f.BaseLayerPath = "/layers/paketo-buildpacks_liberty/base"
```
Comment (dmikusa-pivotal) [inline helper/linker.go:118]: Same as the previous coment.

I don't like that the path is hard coded. We should rely on that env variable being set or fail. The buildpack should always set it, or something is really wrong and we should be failing.

File: openliberty/base_test.go [hunk 6]
```diff
@@ -4,7 +4,7 @@ import (
 	"github.com/buildpacks/libcnb"
 	"github.com/paketo-buildpacks/libpak"
 	"github.com/paketo-buildpacks/libpak/bard"
-	"github.com/paketo-buildpacks/open-liberty/openliberty"
+	"github.com/paketo-buildpacks/liberty/openliberty"
```
Comment (dmikusa-pivotal) [inline openliberty/base_test.go:7]:
```suggestion
	"github.com/paketo-buildpacks/liberty/liberty"
```

File: openliberty/build.go [hunk 7]
```diff
@@ -71,8 +71,8 @@ func (b Build) Build(context libcnb.BuildContext) (libcnb.BuildResult, error) {
 		return result, nil
 	}
 
-	version, _ := cr.Resolve("BP_OPENLIBERTY_VERSION")
-	profile, _ := cr.Resolve("BP_OPENLIBERTY_PROFILE")
+	version, _ := cr.Resolve("BP_LIBERTY_VERSION")
+	profile, _ := cr.Resolve("BP_LIBERTY_PROFILE")
 
 	dep, err := dr.Resolve(fmt.Sprintf("open-liberty-runtime-%s", profile), version)
```
Comment (dmikusa-pivotal) [inline openliberty/build.go:77]: I think this needs to change. It should match the id of the dependencies configured in buildpack.toml.

File: openliberty/build_test.go [hunk 8]
```diff
@@ -28,7 +28,7 @@ import (
 	"github.com/buildpacks/libcnb"
 	. "github.com/onsi/gomega"
 	"github.com/paketo-buildpacks/libpak/bard"
-	"github.com/paketo-buildpacks/open-liberty/openliberty"
+	"github.com/paketo-buildpacks/liberty/openliberty"
```
Comment (dmikusa-pivotal) [inline openliberty/build_test.go:31]:
```suggestion
	"github.com/paketo-buildpacks/liberty/liberty"
```

File: openliberty/detect_test.go [hunk 9]
```diff
@@ -23,7 +23,7 @@ import (
 	"testing"
 
 	"github.com/buildpacks/libcnb"
-	"github.com/paketo-buildpacks/open-liberty/openliberty"
+	"github.com/paketo-buildpacks/liberty/openliberty"
```
Comment (dmikusa-pivotal) [inline openliberty/detect_test.go:26]:
```suggestion
	"github.com/paketo-buildpacks/liberty/liberty"
```

File: openliberty/distribution_test.go [hunk 10]
```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/buildpacks/libcnb"
 	"github.com/paketo-buildpacks/libpak"
 	"github.com/paketo-buildpacks/libpak/bard"
-	"github.com/paketo-buildpacks/open-liberty/openliberty"
+	"github.com/paketo-buildpacks/liberty/openliberty"
```
Comment (dmikusa-pivotal) [inline openliberty/distribution_test.go:29]:
```suggestion
	"github.com/paketo-buildpacks/liberty/liberty"
```

File: openliberty/feature_test.go [hunk 11]
```diff
@@ -4,7 +4,7 @@ import (
 	"encoding/xml"
 	. "github.com/onsi/gomega"
 	"github.com/paketo-buildpacks/libpak/bard"
-	"github.com/paketo-buildpacks/open-liberty/openliberty"
+	"github.com/paketo-buildpacks/liberty/openliberty"
```
Comment (dmikusa-pivotal) [inline openliberty/feature_test.go:7]:
```suggestion
	"github.com/paketo-buildpacks/liberty/liberty"
```

**Review by dmikusa-pivotal — CHANGES_REQUESTED** [review]
You would also need to rename the top-level directory `openliberty` to `liberty`.

**Comment by dmikusa-pivotal** [comment]
There are also a number of places where we use `OL` short for open-liberty. We should change those to just `liberty`. I think it's mainly in env variable names like `BPI_OL_...`.

File: openliberty/build.go (hunk 7 above)
Comment (kevin-ortega) [reply openliberty/build.go:77]: Yes, this will need to change when we add websphere liberty in buildpack.toml.  For now, we just have open-liberty.

**Comment by kevin-ortega** [comment]
@dmikusa-pivotal I don't think the 2 failing actions are something I can fix?

**Review by dmikusa-pivotal — APPROVED** [review]
OK, tests are passing. I'm going to merge this and rename the repo. If we've missed anything, we can adjust in a subsequent PR. I think it'll be easier to validate after the repo is renamed.

**Pull request merged**